packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -3627,8 +3635,20 @@ export class ProjectService extends BaseService {
             throw new ForbiddenError();
         }
         try {
-            const charts = await this.savedChartModel.find({ projectUuid });
-            return charts.length > 0;
+            const charts = await this.contentModel.findSummaryContents(
+                {
+                    projectUuids: [projectUuid],
+                    contentTypes: [ContentType.CHART],
+                    chart: {
+                        sources: [ChartSourceType.DBT_EXPLORE],
```

Comment:
Previously we also only checked for charts in `SavedChartModel` so this is the same check but we can change this to be looking for all `ChartSourceType` since they can all be added to dashboards (sql_runner, semantic_viewer)